src/io/show/ModelWithResult.jl

```diff
@@ -0,0 +1,5 @@
+function Base.show(io::Base.IO, ::MIME"text/plain", res::ModelWithResult)
+    println(io, "A ModelWithResult composed of:")
+    println(io, "model: $(typeof(res.model))")
+    println(io, "result: $(typeof(res.result))")
+end
```

Comment:
show should be preferably as parseable as possible, multiline is discouraged.

Let's go for this:
```suggestion
function Base.show(io::Base.IO, ::MIME"text/plain", res::ModelWithResult{T}) where T
    println(io, "ModelWithResult{$T}($(res.model), ...)")
end
```